### PR TITLE
Add tables `aws_inspector2_coverage`, `aws_inspector2_coverage_statistics`, `aws_inspector2_finding`, and `aws_inspector2_member` [WIP]

### LINKED ISSUE #1643: Add tables for inspector2 findings
Unable to get results from inspector queries when using inspector2. Need tables created so I can run queries against number of vulnerabilities detected that are High, Medium and informational as well as id, arn, asset type and numerical severity for a start.

### PR DESCRIPTION
**Pull request by jaredreisinger-drizly**
_**NOTE:** This is marked as a draft PR until documentation and tests are completed, but I wanted to get eyes on this to make sure I was going down the right track.  I'm also not sure how to add or run tests using the TF/SQL/JSON system._

Nominally fixes #1643

# Integration test logs (TODO)
<details>
  <summary>Logs</summary>

```
Add passing integration test logs here
```
</details>

# Example query results (TODO)
<details>
  <summary>Results</summary>

```
Add example SQL query results here (please include the input queries as well)
```
</details>

### CODE REVIEW

File: aws/service.go [hunk 1]
```diff
@@ -763,6 +764,18 @@ func InspectorClient(ctx context.Context, d *plugin.QueryData) (*inspector.Clien
 	return inspector.NewFromConfig(*cfg), nil
 }
 
+func Inspector2Client(ctx context.Context, d *plugin.QueryData) (*inspector2.Client, error) {
+	// cfg, err := getClientForQuerySupportedRegion(ctx, d, inspector2Endpoint.EndpointsID)
+	cfg, err := getClientForDefaultRegion(ctx, d)
```
Comment (jaredreisinger-drizly) [inline aws/service.go:769]: Oops!  I need to reinstate the region-aware client here!

**Comment by cbruno10** [comment]
@jaredreisinger-drizly Sorry for the delay in review, I should get a chance later this week!

**Comment by seanthebear** [comment]
Any update on this?

**Comment by misraved** [comment]
Thanks @jaredreisinger-drizly for the great work on the tables. Apologies for the delay, but I finally got a chance to check the structure and the design of the tables.

I think from an API and design perspective, the changes look good. Could you please add the table docs and the example queries so that we could proceed with the next steps of the review process?

From the example queries perspective we are definitely curious to see how the optional quals are handled and what valuable information the columns return. Please do let us know if you have questions. Thanks 👍.

**Comment by seanthebear** [comment]
Hi all - Looks like this is dead in the water. I'n afraid I haven't the skills to pick this up and take it forward. Is there a way to mark this as abandoned? Is there anyone else working on indpector2 integration?

**Comment by misraved** [comment]
Apologies @seanthebear for the delay. 

We will start working on it and look to get it out by the end of this week or early next week.

The structure looks good, need to test it out in detail and add the table docs 👍.

**Comment by seanthebear** [comment]
Ooh thank you sooo much. This will make replacing a custom built horrific dashboard using way too many languages and APIs so much easier. Last piece in the puzzle! Very much appreciated.

**Comment by seanthebear** [comment]
Great to see this has been done. How long until the branch gets merged/new version released etc.

**Pull request merged**